cluster/charts/rook/templates/wait-on-resources-job.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    helm.sh/hook: post-install
+    "helm.sh/hook-delete-policy": hook-succeeded
+  labels:
+    app: rook-operator
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+  name: rook-operator-get-resources
+spec:
+  template:
+    metadata:
+      labels:
+        app: rook-operator
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+      name: rook-operator-get-resources-get-crd
+    spec:
+      containers:
+        - name: hyperkube
+          image: "{{ .Values.hyperkube.repository }}:{{ .Values.hyperkube.tag }}"
+          imagePullPolicy: "{{ .Values.hyperkube.pullPolicy }}"
+          command:
+            - ./kubectl
+            - get
+            {{- if ge .Capabilities.KubeVersion.Minor "7" }}
```

Comment:
should the major version be factored in here too, so that this check doesn't break when kubernetes goes to 2.0?